src/transformers/pipelines/automatic_speech_recognition.py

```diff
@@ -144,7 +144,18 @@ def __init__(self, feature_extractor: Union["SequenceFeatureExtractor", str], *a
             raise ValueError("The AutomaticSpeechRecognitionPipeline is only available in PyTorch.")
 
         self.check_model_type(dict(MODEL_FOR_SPEECH_SEQ_2_SEQ_MAPPING.items() + MODEL_FOR_CTC_MAPPING.items()))
-        self.is_ctc = self.model.__class__ in MODEL_FOR_CTC_MAPPING.values()
```

Comment:
@Narsil - I think you accidently removed this no? It's weird that the tests didn't fail as I think the ASR pipelines don't work on master at the moment

Reply:
No this was replaced by `self.type` ({`ctc`, `ctc_with_lm` , `seq2seq`)